cob_openni2_tracker/launch/create_nite2_link.bash

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
```

Comment:
Is this file replaced by:
list(APPEND @PROJECT_NAME@_LIBRARIES "${@PROJECT_NAME@_PREFIX}/@CATKIN_PACKAGE_LIB_DESTINATION@/libNiTE2.so")